cancermuts/datasources.py

```diff
@@ -1200,13 +1218,21 @@ def __init__(self, version='2.1'):
         self._gnomad_endpoint = 'https://gnomad.broadinstitute.org/api/'
         self._cache = {}
         self._supported_metadata = {'gnomad_exome_allele_frequency' : self._get_exome_allele_freq,
-                                    'gnomad_genome_allele_frequency' : self._get_genome_allele_freq}
+                                    'gnomad_genome_allele_frequency' : self._get_genome_allele_freq,
+                                    'gnomad_popmax_exome_allele_frequency' : self._get_popmax_exome_allele_freq,
+                                    'gnomad_popmax_genome_allele_frequency' : self._get_popmax_genome_allele_freq}
 
         gnomADExomeAlleleFrequency.set_version_in_desc(self._version_str[version])
         gnomADGenomeAlleleFrequency.set_version_in_desc(self._version_str[version])
+        gnomADPopmaxExomeAlleleFrequency.set_version_in_desc(self._version_str[version])
+        gnomADPopmaxGenomeAlleleFrequency.set_version_in_desc(self._version_str[version])
 
     def add_metadata(self, sequence, md_type=['gnomad_exome_allele_frequency'], use_alias=None):
-        
+        for md in md_type:
+            if md not in self._version_metadata_compatability[self._gnomad_version]:
+                self.log.error(f"The  metadata type '{md}' is not compatible with the gnomAD version. Compatible metadata are {self._version_metadata_compatability[self._gnomad_version]}")
+                raise ValueError
```

Comment:
can we also have the error as exception message? So it's easier to debug for who might encounter it